enterprise/frontend/src/embedding-sdk/lib/theme/embedding-color-palette.ts

```diff
@@ -2,26 +2,33 @@ import type { MetabaseColor, MetabaseColors } from "embedding-sdk/types/theme";
 import { colors } from "metabase/lib/colors";
 import type { ColorName, ColorPalette } from "metabase/lib/colors/types";
 
-export const SDK_TO_MAIN_APP_COLORS_MAPPING: Record<MetabaseColor, ColorName> =
-  {
-    brand: "brand",
-    border: "border",
-    filter: "filter",
-    summarize: "summarize",
-    "text-primary": "text-dark",
-    "text-secondary": "text-medium",
-    "text-tertiary": "text-light",
-    background: "bg-white",
-    "background-hover": "bg-light",
-    shadow: "shadow",
+import { getEmbeddingChartColors } from "./get-embedding-chart-colors";
 
-    // positive: "success",
-    // negative: "danger",
-    // warning: "warning",
+// Exclude sdk colors that are not 1:1 mappable.
+export type MappableSdkColor = Exclude<MetabaseColor, "charts">;
```

Comment:
Is this the case for colors that are stored as hex values instead of indexes? Can you add some explanation in the comment?

Reply:
@npretto this is only for the `charts` color at this moment, as that is using a custom object. Good point, I'll clarify in the comments a bit on what the "1:1 mappable" part means!

```ts
charts: ["#2d2d30", { base: ..., shade: ..., tint: ... }]
```